presidio-analyzer/presidio_analyzer/pattern_recognizer.py

```diff
@@ -37,9 +39,9 @@ def __init__(
         deny_list: List[str] = None,
         context: List[str] = None,
         deny_list_score: float = 1.0,
+        global_regex_flags: Optional[int] = re.DOTALL | re.MULTILINE | re.IGNORECASE,
```

Comment:
This is pretty bad. You just change the default value from `re.DOTALL | re.MULTILINE` to `re.DOTALL | re.MULTILINE | re.IGNORECASE`.

This affect us as a client of this library.

Reply:
@ducquangkstn thanks for the feedback. This change allows you to have more customizability with regards to regex flags. Is this blocking you in any way?

Reply:
>  Is this blocking you in any way?

Actually, no. It just took me a while to figure out why the behavior is changed when pumping presidio version.

We (my company) are lucky that we have some unit tests. Not sure about other ppl.